pkg/monitortests/network/disruptionpodnetwork/monitortest.go

```diff
@@ -145,7 +147,23 @@ func (pna *podNetworkAvalibility) StartCollection(ctx context.Context, adminREST
 		return err
 	}
 
-	if _, err := pna.kubeClient.CoreV1().Services(pna.namespaceName).Create(context.Background(), podNetworkTargetService, metav1.CreateOptions{}); err != nil {
+	service, err := pna.kubeClient.CoreV1().Services(pna.namespaceName).Create(context.Background(), podNetworkTargetService, metav1.CreateOptions{})
+	if err != nil {
+		return err
+	}
+
+	podNetworkServicePollerDep.Spec.Replicas = &numNodes
+	podNetworkServicePollerDep.Spec.Template.Spec.Containers[0].Env = []corev1.EnvVar{
+		corev1.EnvVar{
+			Name:  "SERVICE_CLUSTER_IP",
+			Value: service.Spec.ClusterIP,
+		},
+		corev1.EnvVar{
+			Name:  "PORT",
+			Value: fmt.Sprintf("%d", service.Spec.Ports[0].Port),
+		},
+	}
+	if _, err = pna.kubeClient.AppsV1().Deployments(pna.namespaceName).Create(context.Background(), podNetworkServicePollerDep, metav1.CreateOptions{}); err != nil {
```

Comment:
need to CollectData.  I refactored slightly to make this easier in https://github.com/openshift/origin/pull/28198 .  You might want to pull that in and use t he factorization.

Reply:
I changed the network.openshift.io/disruption-actor for the service-poller deployment and added the type of connection to the CollectData() is there something else that needs to be done to collect data?